tests/dp_computations_test.py

```diff
@@ -172,6 +172,68 @@ def test_compute_dp_sum(self):
         self._test_gaussian_noise(results, 10, params.eps, params.delta,
                                   l2_sensitivity)
 
+    def test_equally_split_budget(self):
+        # The number of mechanisms must be bigger than 0.
+        with self.assertRaises(ValueError):
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 0)
+
+        # Only one mechanism.
+        self.assertEqual(
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 1),
+            [(0.5, 1e-10)])
+
+        # Multiple mechanisms.
+        expected_budgets = [(0.5 / 5, 1e-10 / 5) for _ in range(4)]
+        expected_budgets.append((0.5 - 4 * (0.5 / 5), 1e-10 - 4 * (1e-10 / 5)))
+
+        self.assertEqual(
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 5),
+            expected_budgets
+        )
+
+    def test_compute_dp_mean(self):
+        params = pipeline_dp.dp_computations.MeanVarParams(
+            eps=0.5,
+            delta=1e-10,
+            low=1,
+            high=20,
+            max_partitions_contributed=1,
+            max_contributions_per_partition=1,
+            noise_kind=pipeline_dp.NoiseKind.LAPLACE)
+
+        (sum_eps, sum_delta), (count_eps,
+                               count_delta) = pipeline_dp.dp_computations.equally_split_budget(
```

Comment:
this line looks pretty long. Can you try run again `make format`?

Reply:
Sure :)